docs/layouts.rst

```diff
@@ -169,13 +165,7 @@ Renders to::
         'form_field_2'
     )
 
-
-Uni-form layout objects
-~~~~~~~~~~~~~~~~~~~~~~~
-
-These ones live in module ``crispy_forms.layout``. Probably in the future they will be moved out to a ``uni_form`` module:
-
-- **ButtonHolder**: It wraps fields in a ``<div class=”buttonHolder”>``, which uni-form positions in a nice way. This is where form's submit buttons go in uni-form::
+- **ButtonHolder**: It wraps fields in a ``<div class=”buttonHolder”>``, this is a legacy layout object from the ``uni-form`` template pack::
```

Comment:
The docs recommend using `FormActons` but these templates exist in the bootstrap template packs. Therefore folk will be using them and therefore think it's ok to keep these even if we acknowledge they are not the recommended approach. I don't think there is a need to introduce a potential breaking change at this stage for this.